### PR TITLE
Hide docker, puppet, and vagrant section for ewcdocs

### PR DESCRIPTION
**Pull request by m4dcoder**
Move the docker, puppet, and vagrant install topics to include files and then use include directives to include these files when building community docs.

### CODE REVIEW

File: docs/source/install/puppet.rst [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
 The ``stackstorm-st2`` Puppet module is available on Puppet Forge:
 `stackstorm-st2 <https://forge.puppet.com/stackstorm/st2>`_
 
 Source code for the module is available as a GitHub repo:
 `StackStorm/puppet-st2 <https://github.com/stackstorm/puppet-st2/>`_
 
+.. note::
+
+    This puppet module only supports the open source version of StackStorm. Please contact
+    enterprise support if you want more info on using puppet module to install
```
Comment (LindsayHill) [inline docs/source/install/puppet.rst]: Should provide some link for users here

**Comment by nmaludy** [comment]
FYI, I'm planning on supporting EWC install in Puppet sometime in the future

**Comment by m4dcoder** [comment]
@nmaludy We can update this when it happens.

File: docs/source/install/puppet.rst (hunk 1 above)
Comment (m4dcoder) [reply docs/source/install/puppet.rst]: Added link

File: docs/source/install/vagrant.rst [hunk 2]
```diff
@@ -6,6 +6,12 @@ It's already pre-installed, configured and tested virtual machine image and so s
 time-consuming installation and configuration steps. Perfect for a quick platform overview,
 testing, demo or even using StackStorm in isolated from the internet air-gapped systems.
 
+.. note::
+
+    The virtual machine image only supports the open source version of StackStorm. Please contact
+    `StackStorm Support <support@stackstorm.com>`_ and let us know if you are interested in an
+    Extreme Workflow Composer (EWC) image or VMWare-compatible virtual appliance.
```
Comment (armab) [inline docs/source/install/vagrant.rst]: This block was under the `Virtual Appliance / OVA` section where `EWC` artifacts mention makes sense.
By moving it close to `Vagrant` section which is a pure OSS-solution adds additional level of confusion.

Comment (m4dcoder) [reply docs/source/install/vagrant.rst]: Ok. Please feel free to update it. Thanks.

**Pull request merged**